azkaban-common/src/main/java/azkaban/utils/FileIOUtils.java

```diff
@@ -84,9 +87,10 @@ public static String getSourcePathFromClass(Class<?> containedClass) {
   }
 
   /**
-   * Run a unix command that will symlink files, and recurse into directories.
+   * Run a unix command that will hard link files and recurse into directories.
    */
-  public static void createDeepSymlink(File sourceDir, File destDir)
+
+  public static void createHardDeeplink(File sourceDir, File destDir)
```

Comment:
I'm kinda confused here. 
Are we creating-a-deep-hardlink or creating-a-hard-deeplink? :|